Frequently_used_code/ARD_overpass_predictor.ipynb

```diff
@@ -0,0 +1,1677 @@
+{
```

Comment:
<p>If there's no reason not to use the more accurate timestamps, let's use those.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/736/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>